doc/source/release-notes/Percona-Server-5.6.30-76.3.rst

```diff
@@ -0,0 +1,50 @@
+.. rn:: 5.6.30-76.3
+
+==============================
+ |Percona Server| 5.6.30-76.3 
+==============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.30-76.3 on May 25th, 2016 (Downloads are available `here <http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.30-76.3/>`_ and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.30 <http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-30.html>`_, including all the bug fixes in it, |Percona Server| 5.6.30-76.3 is the current GA release in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.30-76.3 milestone at Launchpad <https://launchpad.net/percona-server/+milestone/5.6.30-76.3>`_.
+
+Bugs Fixed
+==========
+ 
+ When :ref:`tokudb_read_free_replication` was enabled for |TokuDB| and there was no explicit primary key for the replicated |TokuDB| table there could be duplicated records in the table on update operation. The fix disables :ref:`tokudb_read_free_replication` for tables without explicit primary key and does rows lookup for ``UPDATE`` and ``DELETE`` binary log events and issues warning. Bug fixed :bug:`1536663` (:tokubug:`950`).
```

Comment:
It links to https://www.percona.com/doc/percona-server/5.6/tokudb/using_tokudb.html#read-free-replication